### DISCUSSION TITLE
怎么编译打包 发布在国内服务器上

### DISCUSSION THREAD

**Discussion by china-LY** [imsyy/DailyHotApi#19] [Q&A] [answered]
另外问下怎么编译打包  发布在国内服务器上

**Comment by imsyy** [comment, marked as answer]
只要服务器有 `node` 环境就行，`npm install` 安装依赖，`npm start` 运行，然后使用 `Supervisord` 或者其他方式进行进程守护就行 （ 宝塔可以一键部署 ）

**Reply by nljie1103** [reply]
如何通过Supervisord守护进程，为什么我的自动会停止

**Reply by nljie1103** [reply]
运行目录是主目录吗，启动命令是pnpm start吗
![image](https://github.com/imsyy/DailyHotApi/assets/59471244/c7b609f2-8468-490b-93d2-7ba019b09a7a)

**Comment by china-LY** [comment]
我是直接用的pm2 运行的，宝塔面板 上传整个目录 然后执行 pm2 start好像

nljie1183 ***@***.***> 于2024年1月9日周二 22:39写道：

**Comment by imsyy** [comment]
@china-LY @nljie1183 宝塔建议使用 Docker 部署

**Comment by china-LY** [comment]
docker只是方便些，node部署可以自己随时修改代码

底层用户 ***@***.***> 于2024年1月11日周四 11:49写道：
